src/redux/actions.js

```diff
@@ -78,10 +95,27 @@ export function loadRestaurants() {
 
 export function loadRestaurant({ restaurantId }) {
   return async (dispatch) => {
-    dispatch(setRestaurant(null));
-
+    dispatch(setRestaurant(null)); // 통신하는 동안 빈페이지 보여주기
     const restaurant = await fetchRestaurant({ restaurantId });
-
     dispatch(setRestaurant(restaurant));
   };
 }
+
+export function requestLogin() {
+  return async (dispatch, getState) => {
+    // post email, password
+    const {
+      loginFileds: { email, password },
+    } = getState();
+    const accessToken = await postLogin(email, password);
```

Comment:
문의주셨던 내용 확인해보니 함수 호출시 파라미터를 잘못 넘기고 있어서 발생한 것 같아요.

아래와 같이 처리하면 올바르게 API 호출되네요 ㅎㅎ 

```suggestion
    const accessToken = await postLogin({ email, password });
```

함수 호출시 파라미터를 어떤 방식으로 넘기는지 항상 살피며 작업하시면 좋습니다 😄